src/index.ts

```diff
@@ -92,8 +86,15 @@ export class CfClient {
     console.log(identifier, target);
     return defaultValue;
   }
+
+  close(): void {
+    this.pollProcessor.close();
+    if (this.options.enableStream) {
+      this.streamProcessor.close();
+    }
+  }
 }
 
 // need to setup example project and remove this later
-const client = new CfClient('sdk key');
+const client = new CfClient('1c100d25-4c3f-487b-b198-3b3d01df5794');
```

Comment:
Would it make sense to extract this to a separate file that doesn't get checked in?

Reply:
Yes, ticket: https://harness.atlassian.net/secure/RapidBoard.jspa?rapidView=98&projectKey=FFM&modal=detail&selectedIssue=FFM-1351